exampleSite/content/_index/items-1/react.md

```diff
@@ -0,0 +1,7 @@
++++
```

Comment:
Would make react support an "item" fragment instead. Big enough feature and the others could be reduced.

Reply:
Wanted to have item fragments for each of the features except for fragments. Then we could just link each column in items fragments to the item fragments and for the fragments column, link it to the fragments list.

Reply:
Ah ok. Then happy to keep it. As a placeholder for the bootstrap column we can use "open source", if that makes sense.

Reply:
Will do another review after v0.13.0 is released, but I think it should be fine with pipelines merged.